save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsView.kt

```diff
@@ -120,7 +133,9 @@ abstract class UserSettingsView : AbstractView<UserSettingsProps, UserSettingsVi
                 userInfo = user
                 image = ImageInfo(user?.avatar)
                 userInfo?.let { updateFieldsMap(it) }
-                selfOrganizationWithUserList = organizationDtos
+                selfOrganizationWithUserList = organizationDtos.filter { it.organization.status == OrganizationStatus.CREATED }
```

Comment:
why do we need 3 variables? can we store everything in a single variable and apply different logic according to `status`?

Reply:
Thanks, I'm thinking of doing it in #1537